plugins/emc/emc_vnx_file_

```diff
@@ -0,0 +1,516 @@
+ Make a configuration file at /etc/munin/plugin-conf.d/emc_vnx_file_stats_VNX5300
+ 
+ [emc_vnx_file_*]
+ user munin							# SSH Client local user
+ env.username operator1						# Remote user with Operator role
+ env.cs_addr 192.168.1.1 192.168.1.2				# Control Stations addresses
+ env.nas_servers server_2 server_3				# This is the default value and can be omitteda
+
+=head1 HISTORY
+
+ 08.11.2016 - First Release
+ 17.11.2016 - NFSv4 support, Memory section
+ 16.12.2016 - Merged "NFS" and "Datamover Stats" plugins
+ 26.12.2016 - Compatibility with Munin coding style
+
+=cut
+
+export LANG=C
+
+ssh_check() {
+	ssh -q $username@$1 "/nasmcd/sbin/getreason | grep -w slot_\`/nasmcd/sbin/t2slot\` | cut -d- -f1"
+}
+
+check_conf () {
+	if [ -z "$username" ]; then
+		echo "No username ('username' environment variable)!"
+		return 1
+	fi
+
+	if [ -z "$cs_addr" ]; then
+		echo "No control station addresses ('cs_addr' environment variable)!"
+		return 1
+	fi
+
+	#Choosing Cotrol Station. Code have to be "10"
+	for CS in $cs_addr; do
+		if [[ "10" -eq "$(ssh_check $CS)" ]]; then
+	#		echo "$CS is Primary"
+			PRIMARY_CS=$CS
+			break
+		fi
+	done
+
+	if [ -z "$PRIMARY_CS" ]; then
+		echo "No alive primary Control Station from list \"$cs_addr\"";
+		return 1
+	fi
+	return 0
+}
+
+if [ "$1" = "autoconf" ]; then
+	check_conf_ans=$(check_conf)
+	if [ $? -eq 0 ]; then
+		echo "yes"
+	else
+		echo "no ($check_conf_ans)"
+	fi
+	exit 0
+fi
+
+if [ "$1" = "suggest" ]; then
+	echo "nfs_stats"
+	echo "basicdm_stats"
+	exit 0;
+fi
+
+STATSTYPE=$(echo "${0##*/}" | cut -d _ -f 1-5)
+if [ "$STATSTYPE" = "emc_vnx_file_nfs_stats" ]; then STATSTYPE=NFS; 
+elif [ "$STATSTYPE" = "emc_vnx_file_basicdm_stats" ]; then STATSTYPE=BASICDM;
+else echo "Do not know what to do. Name the plugin as 'emc_vnx_file_nfs_stats_<HOSTNAME>' or 'emc_vnx_file_basicdm_stats_<HOSTNAME>'"; exit 1; fi
+# echo $STATSTYPE
+
+TARGET=$(echo "${0##*/}" | cut -d _ -f 6)
+: ${nas_servers:="server_2 server_3"}
+
+check_conf
+if [[ $? -eq 1 ]]; then
+	exit 1;
+fi
+
+SSH="ssh -q $username@$PRIMARY_CS . /home/operator1/.bash_profile; "
+
+echo "host_name ${TARGET}"
+
+if [ "$1" = "config" ] ; then
+	for server in $nas_servers; do
+		$SSH nas_server -i $server | grep -q 'type *= nas'
+		if [ "$?" != 0 ]  ; then continue; fi
+		nas_server_ok=TRUE
+		
+		if [ $STATSTYPE = "BASICDM" ] ; then
+			cat <<-EOF 
+			multigraph emc_vnx_cpu_percent
+			graph_title EMC VNX 5300 Datamover CPU Util %
+			graph_vlabel %
+			graph_category cpu
+			graph_scale no
+			graph_args --upper-limit 100 -l 0
+			${server}_cpuutil.min 0
+			${server}_cpuutil.label $server CPU util. in %.
+
+			multigraph emc_vnx_network_kib
+			graph_title EMC VNX 5300 Datamover Network bytes over all interfaces
+			graph_vlabel B/s recv. (-) / sent (+)
+			graph_category network
+			graph_args --base 1000
+			${server}_net_in.graph no
+			${server}_net_in.label none
+			${server}_net_in.cdef ${server}_net_in,1000,*
```

Comment:
Just to avoid any misunderstandings: @steveschnepp referred to "50k kbytes" as the (unwanted) result when kbytes are stored by munin (instead of bytes) and rrdtool scales the value representation (e.g. `50000` -> `50k`).

Thus in your case you should make sure that you store the base SI unit (here: bytes), .e.g.:
```echo "${fieldname}.value $((some_value * 1000))```
Then you can remove the `cdef`s and any scaling related options, since munin and rrdtool will just do the magic for you.